__init__.py

```diff
@@ -3,4 +3,5 @@
 from jax.base.environments.jax_environment import JaxEnvironment
 from jax.base.environments.sinara_environment import SinaraEnvironment
 from jax.base.experiments.jax_experiment import JaxExperiment
+from jax.base.experiments.scan_1d import Scan1D
```

Comment:
Perhaps just Scan?  The most basic version is 1D by default.  Can make clear in the docstring.